types/usercentrics-browser-ui/integration-interfaces.d.ts

```diff
@@ -25,8 +25,13 @@ export interface BaseTCFUserDecision {
     /** Indicates if the user gave consent (true) or denied consent (false) */
     consent?: boolean;
 
-    /** The id of the purpose/vendor the consent decision belongs to */
-    id: number;
+    /**
+     * The id of the purpose/vendor the consent decision belongs to.
+     * Note: Different pages in the UserCentrics docs disagree on the type, so both are allowed:
```

Comment:
Ah, I always misspell this :sweat_smile: 

```suggestion
     * Note: Different pages in the Usercentrics docs disagree on the type, so both are allowed:
```

Reply:
Eagle eye 🔍  No worries, I squash the commits afterwards. So keep updating :D